packages/peregrine/lib/talons/MiniCart/useMiniCart.js

```diff
@@ -0,0 +1,67 @@
+import { useCallback, useMemo } from 'react';
+import { useQuery, useMutation } from '@apollo/react-hooks';
+
+import { useCartContext } from '../../context/cart';
+
+export const useMiniCart = props => {
+    const { queries, mutations } = props;
+    const { shoppingBagQuery } = queries;
+    const { removeItemMutation } = mutations;
+
+    const [{ cartId }] = useCartContext();
+
+    const {
+        data: shoppingBagData,
```

Comment:
The "shopping bag" terminology was nice while it lasted, but we've abandoned it. Probably want to update these names.